frontends/PyCDE/src/pycde/module.py

```diff
@@ -2,7 +2,7 @@
 #  See https://llvm.org/LICENSE.txt for license information.
 #  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-from __future__ import annotations
+from .hacks import attribute_to_var
```

Comment:
This seems to include code from a previous revision.

Reply:
Yeah, I fucked up a rebase really bad.